lab-ben/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "lab-ben",
+  "version": "1.0.0",
+  "description": "Data Storage using Express",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha",
+    "build": "mocha && eslint**",
```

Comment:
Good job doing the build script, but remember to install ``eslint`` as a dev dependency too.